addons/account/wizard/account_move_send.py

```diff
@@ -386,7 +406,11 @@ def _prepare_invoice_pdf_report(self, invoice, invoice_data):
         if invoice.invoice_pdf_report_id:
             return
 
-        content, _report_format = self.env['ir.actions.report']._render('account.account_invoices', invoice.ids)
+        if not self.pdf_template_id.xml_id:
+            # This is meant to generate dummy xml_id if none was found
+            self.pdf_template_id.export_data(['id', 'xml_id'])
```

Comment:
The method is api.model. In this method, you must not access the current record because in some flows, the flow is made without instanciating an account.move.send wizard.
Deal with it in `_process_send_and_print` if `wizard` is there or not. Hide the value on dictionaries or add an extra parameter.

I don't understand what you are doing with `export_data` here. If there is no xml_id, who cares?

Reply:
If there is no xml_id, the method _render() will throw an exception

Reply:
I have removed this call since it is ambiguous
I am raising an error instead if the xml_id is missing